Library/Homebrew/cmd/test-bot.rb

```diff
@@ -428,9 +428,37 @@ def formula formula_name
         return
       end
 
+      # unlink any conflicts
+      formula.conflicts.each do |conflict|
+        confict_formula = Formulary.factory(conflict.name)
+
+        if confict_formula.installed? && confict_formula.linked_keg.exist?
+          test "brew", "unlink", "--force", conflict.name
+        end
```

Comment:
Maybe pull the inside of this loop into either a function or combine the blocks below by getting all the dependency formulae and the conflict formula into a single array.